frontend/src/components/layout/HeaderAuthorized.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import VerifiedIcon from '@mui/icons-material/Verified';
+
+import PATHS from '../../constants/routes';
+import {
+  LayoutHeader, 
+  HeaderDivider, 
+  SpaceHolder,
+  BrandLogo, 
+  HeaderContent, 
+  MyCoursesCounter, 
+  MyCoursesCounterContent, 
+  MyCoursesCounterNumber, 
+  LogOut
+} from './styled'
+import UserDropDownContainer from './UserDropDownContainer'
+
+interface Props {
+  user?: {
+    role?: string,
+    unit?: string,
+    department?: string,
+    group?: string
```

Comment:
Let's create on the `src` level new folder `types`. And let's create a new file inside it `user.ts` and provide user type description here. It will allow us to reuse some of the props.